src/sentry/workflow_engine/handlers/condition/issue_category_handler.py

```diff
@@ -17,7 +17,9 @@ def evaluate_value(job: WorkflowJob, comparison: Any) -> bool:
         except (TypeError, ValueError, KeyError):
             return False
 
-        if group and group.issue_category:
-            return bool(value == group.issue_category)
+        try:
+            issue_category = group.issue_category
+        except ValueError:
```

Comment:
do you want to write a test for this :P

Reply:
I'm not sure it's possible to mock this because of how `issue_category` is written?

https://github.com/getsentry/sentry/blob/40f34ed7d51eff8dc377a634ee4fa59ca3f6ebc9/src/sentry/models/group.py#L1025-L1027

Reply:
nvm added 🤡 https://github.com/getsentry/sentry/pull/83056/commits/e08605c4f55291e2aeeb0867ff99dfed3c0e4d58